src/main/groovy/life/qbic/utils/NanoporeParser.groovy

```diff
@@ -171,6 +177,24 @@ class NanoporeParser {
             return convertedDirectory
         }
 
+        private static Map convertToRelativePaths(Map content, Path root) {
+            content["path"] = toRelativePath(content["path"] as String, root)
+            if (content["children"]) {
+                // Children always contains a map, so convert recursively
+                content["children"] = (content["children"] as List).collect { convertToRelativePaths(it as Map, root) }
+            }
+            return content
+
+        }
+
+        private static String toRelativePath(String path, Path root) {
+            if (root.toString() == path) {
+                return "./"
```

Comment:
You could also set this to be empty. I like `"./"` though.
```suggestion
                return '""'
```

Reply:
Lets leave it like this. I think `./` is a proper relative path annotation

Reply:
aggreed